packages/playground/src/dashboard/dao_view.vue

```diff
@@ -36,28 +36,32 @@
           </v-card>
 
           <v-card class="my-3 pa-5" v-for="(proposal, i) in filteredProposals(tab.content.value)" :key="i">
-            <v-card-title class="pa-0 mb-5" v-if="proposal.action">
+            <v-card-title class="pa-0 mb-5 font-weight-bold" v-if="proposal.action">
               {{ proposal.action }}
             </v-card-title>
-            <!-- <v-card-subtitle class="pb-0">
-              <p class="font-weight-bold">
-                Proposal hash: <span class="text--secondary">{{ proposal.hash }}</span>
-              </p>
-            </v-card-subtitle> -->
+            <v-divider class="mt-1 mb-5 text-red-700" v-if="proposal.action" />
```

Comment:
might be better to just wrap these two components in a div with a single v-if="proposal.action"